lighthouse-core/audits/server-response-time.js

```diff
@@ -11,27 +11,27 @@ const MainResource = require('../computed/main-resource.js');
 
 const UIStrings = {
   /** Title of a diagnostic audit that provides detail on how long it took from starting a request to when the server started responding. This descriptive title is shown to users when the amount is acceptable and no user action is required. */
-  title: 'Server response times are low (TTFB)',
+  title: 'Server response times are low',
   /** Title of a diagnostic audit that provides detail on how long it took from starting a request to when the server started responding. This imperative title is shown to users when there is a significant amount of execution time that could be reduced. */
-  failureTitle: 'Reduce server response times (TTFB)',
+  failureTitle: 'Reduce server response times',
   /** Description of a Lighthouse audit that tells the user *why* they should reduce the amount of time it takes their server to start responding to requests. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
-  description: 'Time To First Byte identifies the time at which your server sends a response.' +
+  description: 'Server Response Time identifies the time at which your server sends a response.' +
```

Comment:
we are going to lose these three strings in the localized files (all the other strings touched in this PR should be good, just description/path changes).

One option: leave the "TTFB" part in these strings and go back and remove it in a follow up after 6.0.

Reply:
i'm okay with being untranslated for these for now. we'll fix in a patch rls

Reply:
```suggestion
  description: 'Keep the server response time for the main document short because all other requests depend on it.' +
```
Or something like that? right now it doesn't really say why, we kinda need to say we only check for the root document now even more than before since "server response time" is such a generic concept I would expect an audit named that to potentially apply to all my XHRs too